status-client-spec.md

```diff
@@ -169,6 +169,9 @@ new ones.
 The current list of static peers is published on https://fleets.status.im/. `eth.beta` is the current
 group of peers the official Status client uses. The others are test networks.
 
+Finally, Waku node addresses can be retrieved by traversing 
```

Comment:
The consequence of this is adding Waku nodes as DNS TXT records to https://fleets.status.im. cc @jakubgs

Reply:
What we currently did in `eth.test` is that we run Waku and Whisper subprotocols on the same nodes. So really no additional discovery is needed to start testing.

Reply:
If we want to use DNS records we can include those in https://fleets.status.im/.

Reply:
@adambabik I was using it as a placeholder we can do it with whatever subdomain.